<?xml version="1.0"?>
<launch>
  <node name="state_estimator" pkg="igvc" type="StateEstimator" output="screen">
    <remap from="state_estimator/gps" to="/fix" />
    <remap from="state_estimator/imu" to="/imu" />

    <param name="gravity_magnitude" type="double" value="9.80511" />
    <param name="accelerometer_noise_sigma" type="double" value="1e-2" />
    <param name="gyroscope_noise_sigma" type="double" value="1e-3" />
    <param name="imu_integration_noise_sigma" type="double" value="1e-4" />

    <param name="accelerometer_bias_noise_sigma" type="double" value="2e-4" />
    <param name="gyroscope_bias_noise_sigma" type="double" value="3e-5" />
    <param name="yaw_noise_sigma" type="double" value="0.085" />

    <param name="gps_noise_sigma" type="double" value="0.5" />
    <param name="optimization_lag" type="double" value="1.0" />
    <param name="imu_frequency" type="double" value="125.0" />

    <param name="prior_pose_orientation_noise_sigma" type="double" value="0.25" />
    <param name="prior_pose_position_noise_sigma" type="double" value="0.1" />
    <param name="prior_velocity_noise_sigma" type="double" value="0.1" />
    <param name="prior_accelerometer_bias_sigma" type="double" value="0.8" />
    <param name="prior_gyroscope_bias_sigma" type="double" value="0.8" />
    <param name="gps_transform_noise_sigma" type="double" value="5e-2" />
  </node>
</launch>
